demos/defender/aws_iot_demo_defender.c

```diff
@@ -299,6 +311,12 @@ int RunDefenderDemo( bool awsIotMqttMode,
             /* Disconnect MQTT */
             IotMqtt_Disconnect( mqttConnection, false );
         }
+        else
+        {
+            status = EXIT_FAILURE;
+            IotLogError( "AwsIotDefender_Start() returned with error status %s.",
+                         AwsIotDefender_strerror( defenderResult ) );
```

Comment:
```suggestion
            IotLogError( "AwsIotDefender_Start() returned error. Status=%s.",
                         AwsIotDefender_strerror( defenderResult ) );
```

Reply:
The original log is not any less understandable.